medical-system/src/components/shared/Sidebar.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from "react-router-dom";
+import { ROUTES } from "../../router/routes";
+
+import {
+  MdDashboard,
+  MdLocalHospital,
+  MdPeople,
+  MdSettings,
+} from "react-icons/md";
+import { FaRegCalendarPlus } from "react-icons/fa";
+import { CgProfile } from "react-icons/cg";
+import { FaUserDoctor } from "react-icons/fa6";
+import { IconType } from "react-icons";
+
+export function Sidebar() {
+  const SIDEBAR_LINKS: {
+    id: number;
+    path: string;
+    name: string;
+    icon?: IconType;
+  }[] = [
+    { id: 1, path: ROUTES.dashboard, name: "Dashboard", icon: MdDashboard },
+    { id: 2, path: ROUTES.patients, name: "Patients", icon: MdPeople },
+    { id: 3, path: ROUTES.doctors, name: "Doctors", icon: FaUserDoctor },
+    {
+      id: 4,
+      path: ROUTES.laboratorist,
+      name: "Laboratorist",
+      icon: MdLocalHospital,
+    },
+    { id: 5, path: ROUTES.calendar, name: "Calendar", icon: FaRegCalendarPlus },
+    { id: 6, path: ROUTES.profile, name: "Profile", icon: CgProfile },
+    { id: 7, path: ROUTES.settings, name: "Settings", icon: MdSettings },
+  ];
+
+  return (
+    <>
+      <div className=" bg-custom-viollet w-1/6 h-screen">
+        <ul className="mt-24">
+          {SIDEBAR_LINKS.map((link) => {
+            return (
+              <li
+                key={link.id}
```

Comment:
Rozumiem że ID to stała wartość, jest to połowicznie dobra sytuacja, ja bym zastosował tutaj może path bardziej.

Poza tym, warto poczytać https://medium.com/react-in-the-real-world/why-you-should-never-ever-use-indices-as-react-keys-8329472170be 
Nie użyłas tutaj indexu, ale po prostu warto wiedzieć:)